packages/fluentui/react-northstar/src/themes/teams/components/Pill/pillStyles.ts

```diff
@@ -0,0 +1,39 @@
+import { ComponentSlotStylesPrepared, ICSSInJSStyle } from '@fluentui/styles';
+import { PillStylesProps } from '../../../../components/Pill/Pill';
+import { PillVariables } from './pillVariables';
+import { getBorderFocusStyles } from '../../getBorderFocusStyles';
+
+export const pillStyles: ComponentSlotStylesPrepared<PillStylesProps, PillVariables> = {
+  root: ({ props: p, variables: v, theme: { siteVariables } }): ICSSInJSStyle => {
+    return {
+      display: 'inline-flex',
+      borderRadius: v.borderRadiusCircular,
```

Comment:
```suggestion
      borderRadius: v.borderRadius,
```

As this is a default radius, we should not use any suffixes